www/_includes/layouts/base.njk

```diff
@@ -28,13 +28,11 @@
     <meta property="twitter:image" content="https://www.snowpack.dev/img/social-4.jpg">
 
     <!-- Global Stylesheets -->
-    {# <link href="https://fonts.googleapis.com/css2?family=Overpass:wght@400;700;900&display=swap" rel="stylesheet"> #}
+    <link href="https://fonts.googleapis.com/css2?family=Overpass:wght@400;700;900&display=swap" rel="stylesheet">
```

Comment:
doh, my bad. I was on that bad internet in Julian and commented this out to speed up page reloads and forgot to revert before committing